<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_commentBtn">
         <property name="text">
          <string>Комментарий</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pointBtn">
         <property name="text">
          <string>Сбор точки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_moveBtn">
         <property name="text">
          <string>Перемещение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_bestFitBtn">
         <property name="text">
          <string>МНК, best fit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_circleBtn">
         <property name="text">
          <string>Окружность</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pointCircleBtn">
         <property name="text">
          <string>Точка по окр.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pointPlaceBtn">
         <property name="text">
          <string>Точка на плос.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_errorPointBtn">
         <property name="text">
          <string>Ошибка тчк.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_errorCircleBtn">
         <property name="text">
          <string>Ошибка окр.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="5">
        <widget class="QPushButton" name="m_allClear">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Удалить все</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="6">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3" colspan="3">
        <widget class="QListWidget" name="m_outputWidget"/>
       </item>
       <item row="3" column="5">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" rowspan="3">
        <widget class="QListWidget" name="m_commandList"/>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="m_startBtn">
         <property name="text">
          <string>Запустить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionCreate_File"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionColor_setting"/>
    <addaction name="action3D"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionGuid"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuView"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionCreate_File">
   <property name="text">
    <string>Create File</string>
   </property>
  </action>
  <action name="actionSet_file_position">
   <property name="text">
    <string>Set file position</string>
   </property>
  </action>
  <action name="actionColor_setting">
   <property name="text">
    <string>Color setting</string>
   </property>
  </action>
  <action name="action3D">
   <property name="text">
    <string>3D</string>
   </property>
  </action>
  <action name="actionGuid">
   <property name="text">
    <string>Guid</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
